scripts/install

```diff
@@ -211,29 +197,17 @@ def main():
     py_version = sys.version_info[:2]
     if py_version in UNSUPPORTED_PYTHON:
         unsupported_python_msg = (
-            "Unsupported Python version detected: Python %s.%s\n"
-            "To continue using this installer you must use Python 2.7, "
-            "Python 3.6 or later.\n"
-            "For more information see the following blog post:\n"
-            "https://aws.amazon.com/blogs/developer/deprecation-of-python-2-6"
-            "-and-python-3-3-in-botocore-boto3-and-the-aws-cli/"
-        )
-        print(unsupported_python_msg % py_version)
+            "Unsupported Python version detected: Python {}.{}\n"
+            "To continue using this installer you must use Python 3.6 "
+            "or later.\n"
+            "For more information see the following blog post: "
+            "https://aws.amazon.com/blogs/developer/announcing-end-"
+            "of-support-for-python-2-7-in-aws-sdk-for-python-and-"
+            "aws-cli-v1/\n"
+        ).format(py_version[0], py_version[1])
+        print(unsupported_python_msg, file=sys.stderr)
         sys.exit(1)
 
-    if py_version in DEPRECATED_PYTHON:
```

Comment:
I'm also wondering if we just keep it around similar to what you did for the boto3 PR: https://github.com/boto/boto3/pull/2922/files#diff-44c325661f26048dc7f7108da58a07c0bb25e4f016a6dfc37bfbcab91ceec4beR66? It is sort of annoying that we have to go back and remember/find/reintroduce this logic when a new version of Python is deprecated. I could go either way on this.

Reply:
Yea I thought the same thing.

Reply:
I waffled on this too. I think I'm fine keeping it as a no-op for now. I think we'd just need to put in place holders to make it a useful template for next time.

Reply:
Yeah I'd even be fine with just commenting out what we have right now for Python2.7 as an example to follow in the future.